# cd /data/openpilot && curl https://raw.githubusercontent.com/eFiniLan/openpilot-patches/master/0.5.11/subaru/for_stock_acc_only_does_not_disable_op_on_gas_or_brake.diff | git apply -v
#
# (Suitable for Stock ACC)
# This patch will disable OP disengagement when gas or brake is pressed.
#

--- selfdrive/car/subaru/interface.py
+++ selfdrive/car/subaru/interface.py
@@ -199,13 +199,6 @@
     if not self.CS.acc_active:
       events.append(create_event('pcmDisable', [ET.USER_DISABLE]))
 
-    # disable on gas pedal rising edge
-    if (ret.gasPressed and not self.gas_pressed_prev):
-      events.append(create_event('pedalPressed', [ET.NO_ENTRY, ET.USER_DISABLE]))
-
-    if ret.gasPressed:
-      events.append(create_event('pedalPressed', [ET.PRE_ENABLE]))
-
     ret.events = events
 
     # update previous brake/gas pressed
